docs/prologue/CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Contributing
```

Comment:
should we remove the doc `Contributing` at the root folder ? Since it's a complete copy. That would mean if we are modifying the contributing doc, we will have to it twice

Reply:
Actually I'll remove it from this change, we can keep as is.